RadixWallet.xcodeproj/project.pbxproj

```diff
@@ -8607,15 +8606,15 @@
 			repositoryURL = "https://github.com/pointfreeco/swift-identified-collections";
 			requirement = {
 				kind = exactVersion;
-				version = 1.0.1;
+				version = 1.0.2;
```

Comment:
I bumped IdentifiedCollections to `1.0.2` as well, containing a bug fix of of `sort` crashing (Gustaf already changed our TX history to sort using Array instead to circumvent this)